### ISSUE TITLE
AttributeError: module 'numpy' has no attribute 'int'.

### ISSUE THREAD

**Issue by kyleosung** [SheffieldML/GPy#1025]
Initializing classes results in AttributeError

Code
```python
from GPy.models.gp_coregionalized_regression import GPCoregionalizedRegression as CO
model = CO(X_list, Y_list,)
```

Error
```
AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

**Comment by ekalosak** [comment]
In progress: https://github.com/SheffieldML/GPy/pull/1017

**Comment by MartinBubel** [comment]
#1017 is fixed but now the import still fails due to some `numpy.testing` import error. So again, "in progress" but now waiting for #1030

**Comment by MartinBubel** [comment]
Hey @kyleosung 
this has been fixed as off #1040 

There is no new release yet but there aren't many issues remaining until we can release a hot fix.

In the meantime, you can install `GPy` from the git dependency at `devel` branch, if you need the fix urgently.

**Comment by MartinBubel** [comment]
I will close your issue for the sake of a better overview on our issue board. If the issue persists, feel free to reopen.
